src/modifiers/handleNewCodeBlock.js

```diff
@@ -20,8 +20,18 @@ const handleNewCodeBlock = editorState => {
     if (language) {
       data.language = language;
     }
-    const editorStateWithCodeBlock = changeCurrentBlockType(editorState, "code-block", "", data);
-    return splitBlockAndChange(editorStateWithCodeBlock, undefined, undefined, false)
+    const editorStateWithCodeBlock = changeCurrentBlockType(
+      editorState,
+      "code-block",
+      "",
+      data
+    );
+    return splitBlockAndChange(
+      editorStateWithCodeBlock,
+      undefined,
+      undefined,
+      false
+    );
   }
```

Comment:
this is just prettier doing its thing

Reply:
Yeah we should just add that as a precommit hook, honestly